mix/qml/StatusPane.qml

```diff
@@ -281,24 +284,15 @@ Rectangle {
 					anchors.rightMargin: 9
 					anchors.verticalCenter: parent.verticalCenter
 					id: debugImg
-					iconSource: "qrc:/qml/img/bugiconinactive.png"
+					iconSource: "qrc:/qml/img/bugiconactive.png"
 					action: debugRunActionIcon
-					states: [
-						State{
-							name: "active"
-							PropertyChanges { target: debugImg; iconSource: "qrc:/qml/img/bugiconactive.png"}
-						}
-					]
 				}
 				Action {
```

Comment:
I think this Action should be removed completely. There is already an action to toggle right pane in Application.qml